tests/SuperfishNodeVisitorTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+require_once dirname(__FILE__).'/../src/BlocktopmenuNode.php';
+require_once dirname(__FILE__).'/../src/BlocktopmenuNodeVisitor.php';
+require_once dirname(__FILE__).'/../src/SuperfishNodeVisitor.php';
+
+class SuperfishNodeVisitorTest extends PHPUnit_Framework_TestCase
+{
+    public function testVisitProductNode()
+    {
+        $visitor = new SuperfishNodeVisitor();
+
+        $node = new BlocktopmenuNode(BlocktopmenuNode::TYPE_PRODUCT, array(
+            'name' => 'Foo',
+            'link' => 'http://prestashop.com',
+        ));
+
+        $node->setSelected(false);
+        $visitor->visit($node);
+        $this->assertEquals((string) $visitor, '<li><a href="http://prestashop.com" title="Foo">Foo</a></li>');
+
+        $visitor->reset();
+        $node->setSelected(true);
+        $visitor->visit($node);
+        $this->assertEquals((string) $visitor, '<li class="sfHover"><a href="http://prestashop.com" title="Foo">Foo</a></li>');
```

Comment:
Order of arguments in assertions should be ["expected" then "actual"](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertEquals), not "actual" then "expected". It's easy to remember, it's reverse alphabetical order ;)